static/js/server_events_dispatch.js

```diff
@@ -86,6 +86,8 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
             if (event.data.authentication_methods !== undefined) {
                 settings_org.populate_auth_methods(event.data.authentication_methods);
             }
+        } else if (event.op === 'update' && event.property === 'notifications_stream_id') {
+            page_params.realm_notifications_stream_id = event.value;
```

Comment:
shouldn't this also call the function to render the notifications stream UI?  That way, changes take effect in all browsers, not just the current one.